packages/docs/developer-resources/start.md

```diff
@@ -2,10 +2,10 @@
 
 The following examples will serve as a guide to understand the Celo SDK:
 
-* The [Hello Celo](walkthroughs/hellocelo.md) walkthrough will guide you through the basics of connecting to Celo networks using ContractKit.
+* The [Sending CELO](walkthroughs/hellocelo.md) walkthrough will guide you through the basics of connecting to Celo networks using ContractKit.
 
-* The [Hello Contracts](walkthroughs/hellocontracts.md) walkthrough will guide you through the basics of how to deploy your own smart contracts on Celo networks.
+* The [Deploy a Contract (local node)](walkthroughs/hellocontracts.md) walkthrough will guide you through the basics of how to deploy your own smart contracts on Celo networks.
 
-* The [Hello Contracts (Truffle + Remote Node)](walkthroughs/hello-contract-remote-node.md) demonstrates how to deploy smart contracts with [Truffle](https://www.trufflesuite.com/truffle) and [Forno](forno/).
+* The [Deploy a Contract (Forno)](walkthroughs/hello-contract-remote-node.md) demonstrates how to deploy smart contracts with [Truffle](https://www.trufflesuite.com/truffle) and [Forno](forno/).
```

Comment:
`...demonstrates how to deploy smart contracts with [Truffle](https://www.trufflesuite.com/truffle) and the [Forno](forno/) testnet.`

Add more description to what Forno is.